components/clinical/admin-actions/src/atoms/admin-action-description.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react'
+import styled from '@emotion/styled'
+import { Maybe } from '@ltht-react/types'
+
+const StyledDescription = styled.div`
```

Comment:
For consistency, would rather either
a) inline this into `admin-action.tsx` with the rest of the components
b) extract other emotion styled components form `admin-action.tsx` into their own component files